app/javascript/react/components/Map/Markers/Markers.tsx

```diff
@@ -42,15 +48,30 @@ const Markers = ({ points }: Props) => {
     });
   };
 
+  const centerMapOnMarker = (position: LatLngLiteral, key: string) => {
+    if (map) {
+      map.setCenter(position);
+      map.setZoom(ZOOM_FOR_SELECTED_SESSION);
+    }
+    setSelectedMarkerKey(key === selectedMarkerKey ? null : key);
+  };
+
   return (
     <>
-      {points.map((point) => (
+      {sessions.map((session) => (
         <AdvancedMarker
-          position={point}
-          key={point.key}
-          ref={(marker) => setMarkerRef(marker, point.key)}
+          position={session.point}
+          key={session.point.key}
+          ref={(marker) => setMarkerRef(marker, session.point.key)}
         >
-          <span className="tree">🌳</span>
+          <SingleMarker
+            color="#E95F5F"
```

Comment:
"Can we move this colour to a separate colours file?"

Reply:
I didn't want to litter the Colors file because this color is just temporarily mocked and will be changed in the near future :)